tests/insert_reference/insert_reference.expected

```diff
@@ -0,0 +1,2 @@
+Add reference(s):
+* target.html, l3 --> source.ml, l3, 1 line high
```

Comment:
There is a missing newline here.

Reply:
Do I need to add or delete a newline ? Because I wanted to list the references added like this : 
```sh
Add references(s):
* doc, l1 --> source, l2, 3 line high
* ...
* doc_n, ln --> source_n, ln, n line high
```